packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json-private.test.ts

```diff
@@ -0,0 +1,278 @@
+    instanceId: connectorInstanceId3,
+    pluginRegistry: pluginRegistry3,
+    rpcApiHttpHost: rpcApiHttpHostMember3,
+    privateUrl: keys.quorum.member3.privateUrl,
+    logLevel,
+  });
+  t.ok(connector3, "connector3 truthy OK");
+  test.onFinish(() => connector3.shutdown());
+  pluginRegistry3.add(connector3);
+
+  await connector1.onPluginInit();
+  await connector2.onPluginInit();
+  await connector3.onPluginInit();
+
+  const accountKeyPath = path.resolve(
+    __dirname,
+    "../../../../resources/network-files/member1",
+    "accountkey",
+  );
+  const accountKey = JSON.parse(fs.readFileSync(accountKeyPath, "utf8"));
+  const signingAccount = webQJsMember1.eth.accounts.decrypt(accountKey, "");
+
+  const accountAddress = keys.quorum.member1.accountAddress;
+  const txCount = await webQJsMember1.eth.getTransactionCount(
+    `0x${accountAddress}`,
+  );
+
+  const deployRes = await connector1.deployContract({
+    bytecode: HelloWorldContractJson.bytecode,
+    contractName: HelloWorldContractJson.contractName,
+    privateTransactionConfig: {
+      privateFrom: keys.tessera.member1.publicKey,
+      privateFor: [
+        keys.tessera.member1.publicKey,
+        keys.tessera.member2.publicKey,
+        keys.tessera.member3.publicKey,
+      ],
+    },
+    web3SigningCredential: {
+      secret: keys.quorum.member1.privateKey,
+      type: Web3SigningCredentialType.PrivateKeyHex,
+      ethAccount: JSON.stringify(signingAccount),
+    },
+    keychainId: keychain1.getKeychainId(),
+    gas: 3000000,
+    gasPrice: 0,
+    nonce: txCount,
+    value: 0,
+  });
+
+  console.log(deployRes);
+
+  const contractDeployReceipt = await webQJsMember1.eth.getTransactionReceipt(
+    deployRes.transactionReceipt.transactionHash,
+  );
+
+  t.ok(contractDeployReceipt, "contractDeployReceipt truthy OK");
+  const receipt = contractDeployReceipt;
+  const { contractAddress } = receipt;
+  t.comment(`Private contract address: ${contractAddress}`);
+  t.ok(contractAddress, "contractAddress truthy OK");
+
+  {
+    const member1Contract = new webQJsMember1.eth.Contract(
+      HelloWorldContractJson.abi as any,
+      contractAddress,
+    );
+    const mem1Response = await member1Contract.methods.getName().call();
+    t.equal(
+      mem1Response,
+      "CaptainCactus",
+      "member1 getName() equals 'CaptainCactus'",
+    );
+
+    const member2Contract = new webQJsMember2.eth.Contract(
+      HelloWorldContractJson.abi as any,
+      contractAddress,
+    );
+    const mem2Response = await member2Contract.methods.getName().call();
+    t.equal(
+      mem2Response,
+      "CaptainCactus",
+      "member1 getName() equals 'CaptainCactus'",
+    );
+
+    const member3Contract = new webQJsMember3.eth.Contract(
+      HelloWorldContractJson.abi as any,
+      contractAddress,
+    );
+
+    t.comment("member3 should not be able to call contract");
+    t.throws(await member3Contract.methods.getName().call());
```

Comment:
@travis-payne Does this still works if you switch from `throws` to `rejects` ? 

`throws` is for sync functions and I suspect the only thing it does here is asserting that a promise was returned without throwing, but it does not check if said promise would reject or resolve (even with the `await` keyword baked in there - could be wrong though so I'm just asking to double check)

Reply:
> `throws` works here but running `rejects` seems to crash out.

@travis-payne Please attach the logs that it crashes with.

Reply:
@travis-payne The logs you attached look like they were generated by `throws` not `rejects` (based on the call stack) could you please double check if these are the logs for `rejects` crashing for sure?

Reply:
@petermetz yup, the following code chucks the same error:

`    const promise = member3Contract.methods.getName().call();
    await t.rejects(promise,"member3 should not be able to call contract");`

Reply:
@travis-payne Alright, I'll debug it on my machine then. 
Please push a version of the code where it reproduces after a `yarn configure && yarn ts-node packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json-private.test.ts` and then ping me here and I'll take a peek.